doc/content/devices/arduino/api/cayennelpp/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: CayenneLPP
+weight: 800
+source: 'https://github.com/TheThingsNetwork/arduino-device-lib/blob/master/docs/CayenneLPP.md'
+---
+
+The `CayenneLPP` class enables Arduino devices to encode data with the Cayenne Low Power Payload (LPP). [Read more about Cayenne LPP](https://mydevices.com/cayenne/docs/lora/#lora-cayenne-low-power-payload)
```

Comment:
Can you use `more` tags here to limit what's gonna be shown on cards? Otherwise it looks like this:
![Uploading Screenshot from 2023-04-04 20-02-34.png…]()